docs/resources/stack_dependency.md

```diff
@@ -0,0 +1,27 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spacelift_stack_dependency Resource - terraform-provider-spacelift"
+subcategory: ""
+description: |-
+  spacelift_stack_dependency represents a Spacelift stack dependency - a dependency between two stacks. When one stack depends on another, the tracked runs of the stack will not start until the dependent stack is successfully finished.
+---
+
+# spacelift_stack_dependency (Resource)
+
+`spacelift_stack_dependency` represents a Spacelift **stack dependency** - a dependency between two stacks. When one stack depends on another, the tracked runs of the stack will not start until the dependent stack is successfully finished.
```

Comment:
Additionally, changes to the dependency will trigger the dependent.

Reply:
good call, added